accname/name/shadowdom/basic.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/wai-aria/scripts/aria-utils.js"></script>
+
+<label id="label1">
+  <div id="host1"></div>
+</label>
+<button id="labelled1"
+        class="labelled"
+        type="button"
+        aria-labelledby="label1"
+        data-expectedlabel="foo"
+        data-testname="content inside shadow DOM"></button>
```

Comment:
See next comment. This could be:

"aria-labelledby reference to nameFrom: shadow contents with text node"